packages/dialtone-css/lib/build/less/components/skeleton.less

```
@@ -17,38 +17,42 @@
 //  ============================================================================
 //  @   SKELETON CONTAINER
 //  ----------------------------------------------------------------------------
-.skeleton-placeholder {
+
+// The --placeholder-from-color and --placeholder-to-color
+// custom properties can be set on the parent class of the
+// placeholder to control the animation colors.
+.d-skeleton-placeholder {
   --placeholder-from-color: hsla(var(--dt-color-black-900-hsl) / 0.2);
   --placeholder-to-color: hsla(var(--dt-color-black-900-hsl) / 0.1);
```

```suggestion
  --placeholder-from-color: var(--dt-color-surface-bold-opaque);
  --placeholder-to-color: var(--dt-color-surface-moderate-opaque);
```
Actually, maybe they should be here and not even need a fallback anywhere? I haven't tested it, but off top of my head I can't figure a situation where a fallback kicks in.

---

Yeah don't really think we need the fallback... removed it